README.md

```diff
@@ -18,23 +17,67 @@ Additional code examples can be accessed from the following locations:
 - For articles on using DataRobot, feature deep dives, and example workflows, visit [DataRobot Community](https://community.datarobot.com/).
```

Comment:
delete this full bullet
`For articles on using DataRobot, feature deep dives, and example workflows, visit [DataRobot Community](https://community.datarobot.com/). to:`